libraries/AC_Planck/AC_Planck.cpp

```diff
@@ -149,7 +151,7 @@ void AC_Planck::request_alt_change(const float alt)
     0,                //param3
     alt,              //param4
     0,                //param5
-    false);           //param6
+    *reinterpret_cast<float*>(&(max_rates)));           //param6
```

Comment:
This only works if `float` is a 32-bit type within APM (`float` must be the same size as `uint32_t` in order for `reinterpret_cast` not to cause memory issues).  I think its worth triple-checking this.

Reply:
Just checked, it has to be 32 bit floats, but I'll see about some printouts in testing just to be sure